tests/integration/targets/mso_schema_template_bd_subnet/tasks/main.yml

```diff
@@ -119,49 +104,6 @@
         version: 1
     state: present
 
-- name: Add bd 2
-  mso_schema_template_bd:
-    <<: *bd_present
-    schema: '{{ mso_schema | default("ansible_test") }}_2'
-    template: Template2
-    bd: ansible_test_2
-    intersite_bum_traffic: true
-    optimize_wan_bandwidth: true
-    layer2_stretch: true
-    layer2_unknown_unicast: proxy
-    vrf:
-      name: VRF2
-      schema: '{{ mso_schema | default("ansible_test") }}_2'
-      template: Template2
-    dhcp_policy:
-      name: ansible_test
-      version: 1
-      dhcp_option_policy:
-        name: ansible_test_option
-        version: 1
-
-- name: Add bd
-  mso_schema_template_bd: 
-    <<: *bd_present
-    schema: '{{ mso_schema | default("ansible_test") }}'
-    template: Template1
-    bd: ansible_test_3
-    intersite_bum_traffic: true
-    optimize_wan_bandwidth: true
-    layer2_stretch: true
-    layer2_unknown_unicast: proxy
-    vrf:
-      name: VRF
-      schema: '{{ mso_schema | default("ansible_test") }}'
-      template: Template1
-    dhcp_policy:
-      name: ansible_test
-      version: 1
-      dhcp_option_policy:
-        name: ansible_test_option
-        version: 1
-    state: present
-
```

Comment:
Why remove a lot of those test cases? Same for the rest of this file?

Reply:
by mistake changed this